classes/Commands/UpdateCommand.php

```diff
@@ -79,7 +79,7 @@ protected function execute(InputInterface $input, OutputInterface $output): ?int
 
             // in the case of commands containing the update status, it is not necessary to update the configuration
             // also we do not want to repeat the update of the config in the recursive commands
-            if ($input->getOption('data') === null) {
+            if (!$input->hasOption('data')) {
```

Comment:
here the condition to change, we now look if the command has an option, and not if the option is null

Reply:
To revert